nvflare/apis/fl_context.py

```diff
@@ -90,6 +90,9 @@ def set_prop(self, key: str, value, private=True, sticky=True):
                     f"{self._to_string(existing_mask)}, cannot change to {self._to_string(mask)}"
                 )
                 return False
+
+        if sticky:
+            self._sync_sticky()
```

Comment:
Since we are dealing with the sticky prop copy within the set_prop() method, we should handle the single prop directly, instead of calling _sync_sticky(). The "_sync_sticky()" call will loop through all the props in the fl_ctx.